src/schema/index.js

```diff
@@ -99,15 +99,16 @@ type Query {
 
 input TopicFilter {
   OR: [TopicFilter!]
+  txid: String
```

Comment:
need add this in `resolver.js` to tk effect

Reply:
already in there. forgot to add the definition before.